contracts/contracts/token/OUSD.sol

```diff
@@ -32,40 +27,61 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
     );
     event AccountRebasingEnabled(address account);
     event AccountRebasingDisabled(address account);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+    event Approval(
+        address indexed owner,
+        address indexed spender,
+        uint256 value
+    );
 
     enum RebaseOptions {
         NotSet,
-        OptOut,
-        OptIn
+        StdNonRebasing,
+        StdRebasing,
+        YieldDelegationSource,
+        YieldDelegationTarget
     }
 
+    // Add slots to align with deployed OUSD contract
+    uint256[154] private _gap;
     uint256 private constant MAX_SUPPLY = ~uint128(0); // (2^128) - 1
     uint256 public _totalSupply;
     mapping(address => mapping(address => uint256)) private _allowances;
     address public vaultAddress = address(0);
-    mapping(address => uint256) private _creditBalances;
-    uint256 private _rebasingCredits;
+    mapping(address => uint256) internal _creditBalances;
+    uint256 private _rebasingCredits; // Sum of all rebasing credits (_creditBalances for rebasing accounts)
     uint256 private _rebasingCreditsPerToken;
-    // Frozen address/credits are non rebasing (value is held in contracts which
-    // do not receive yield unless they explicitly opt in)
-    uint256 public nonRebasingSupply;
-    mapping(address => uint256) public nonRebasingCreditsPerToken;
+    uint256 public nonRebasingSupply; // All nonrebasing balances
+    mapping(address => uint256) internal alternativeCreditsPerToken;
     mapping(address => RebaseOptions) public rebaseState;
     mapping(address => uint256) public isUpgraded;
```

Comment:
Let's deprecate `isUpgraded`. We aren't using it any more.

Reply:
Should we also change the `creditsBalanceOfHighres` function signature not to return the `isUpgraded`? Do you have a good way to verify if any contract is still calling this function?

Reply:
Let's keep it there. We've recommended people that function. Just the internal storage slot name can change.